### PR TITLE
update homepage, add html template

### PR DESCRIPTION
**Pull request by samanpwbb**

### CODE REVIEW

**Comment by samanpwbb** [comment]
I edited the copy and layout on the home page a little in addition to adding the html template. I found all the `code` styling in the overview section headlines disorienting so I removed them.

File: site/html_template.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+  <title><!-- Your title goes here --></title>
+  <meta charset='utf-8'>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='shortcut icon' href='/path/to/favicon' type='image/x-icon'>
+  <link href='https://api.mapbox.com/mapbox-assembly/v0.2.0/assembly.min.css' rel='stylesheet'>
+  <script async defer src='https://api.mapbox.com/mapbox-assembly/v0.2.0/assembly.js'></script>
+</head>
+<body>
+ <!-- page content goes here -->
```
Comment (davidtheclark) [inline site/html_template.js]: How about `Your page ...`

File: site/html_template.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const htmlTemplate = `<!DOCTYPE html>
+<html lang='en'>
+<head>
+  <title><!-- Your title goes here --></title>
+  <meta charset='utf-8'>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='shortcut icon' href='/path/to/favicon' type='image/x-icon'>
```
Comment (davidtheclark) [inline site/html_template.js]: Should we say `href='Your favicon path goes here'` to make it more clear that this is another fill-in-the-black situation?

File: site/home.js [hunk 3]
```diff
@@ -146,7 +164,7 @@ class Home extends React.Component {
           For dynamic styles, responsive classes (described above) provide some additional flexibility.
           But if you need even more, you should use custom CSS instead of a utility class.
         </p>
-        <h3 className='mt24 txt-bold'>
+        <h3 className='mt48 txt-bold'>
           <code className='txt-code'>is-active</code> applies active states
```
Comment (davidtheclark) [inline site/home.js:168]: Did you intend to remove the code from this? Aren't you disoriented?

**Review by davidtheclark — COMMENTED** [review]
Good changes 🥇

File: site/home.js (hunk 3 above)
Comment (samanpwbb) [reply site/home.js:168]: I couldn't think of a non-awkward way to re-write this one!!!

**Pull request merged**